### DISCUSSION TITLE
When will NVIDIA provide drivers for musl?

### DISCUSSION THREAD

**Discussion by oifj34f34f** [NVIDIA/open-gpu-kernel-modules#474] [Q&A]
Linux is not limited to distributions that use the GNU C library. For example, there is Alpine Linux, which uses musl and is widely used in the container/docker ecosystem and beyond. Some people want to use it, but can't because of driver limitations...

**Comment by arisu3** [comment]
The driver itself does not rely on musl (or glibc) because it is a kernel module and uses the kernel's internal libc. The problem is the userspace components, some of which are closed source.
